### ISSUE TITLE
How About the Flops, fps and parameter of this model?

### ISSUE THREAD

**Issue by huhaoyue** [ClementPinard/SfmLearner-Pytorch#144]
I'm very interested in the flops, fps and parameter quantities of your model. I wonder if I can get your answer?

**Comment by ClementPinard** [comment]
Hi,

this work did not focus on flops or fps, but rather on the learning method, and how to craft a good loss function from auto supervision.

For the record, the network used here is DispNet, from this paper : https://arxiv.org/pdf/1512.02134.pdf , and nothing has been made to test whether a heavier network or a lighter one would have performed better (in accuracy or speed)

You can try this repo for flops https://github.com/sovrasov/flops-counter.pytorch, and for fps, you can have a look at openVINO which gave a shot to on-mobile inference https://docs.openvino.ai/latest/notebooks/201-vision-monodepth-with-output.html
